examples/js/loaders/GLTFLoader.js

```diff
@@ -172,6 +180,14 @@ THREE.GLTFLoader = ( function () {
 
 			} );
 
+			if ( this.lazy ) {
```

Comment:
There is also some pre-processing in `parser.parse()` that still needs to happen here — specifically `parser.markDefs()`.